### PR TITLE
Change containernetworking to c2d variant

### PR DESCRIPTION
**Pull request by mray190**

### CODE REVIEW

**Review by ryannowarm — CHANGES_REQUESTED** [review]
@mray190 , please add more detailed description to commit message:  Why are we making the change?

File: golang-github-containernetworking-plugins/deb/debian/control [hunk 1]
```diff
@@ -15,6 +15,7 @@ Vcs-Browser: https://salsa.debian.org/go-team/packages/golang-github-containerne
 Vcs-Git: https://salsa.debian.org/go-team/packages/golang-github-containernetworking-plugins.git
 XS-Go-Import-Path: github.com/containernetworking/plugins
 Testsuite: autopkgtest-pkg-go
+Provides: containernetworking-plugins
```
Comment (ryannowarm) [inline golang-github-containernetworking-plugins/deb/debian/control:18]: Why are we using `Provides` here and not `Replaces` and `Conflicts`?  Do we even need this provides at all?

**Review by ryannowarm — CHANGES_REQUESTED** [review]

File: golang-github-containernetworking-plugins/deb/debian/control (hunk 1 above)
Comment (mray190) [reply golang-github-containernetworking-plugins/deb/debian/control]: Removed this provides line

**Comment by mray190** [comment]
Commit message was updated as well

**Comment by mray190** [comment]
Dev tested by Yixin! Looks good, ready for re-review and merge

**Review by ryannowarm — APPROVED** [review]

**Pull request merged**